js/dataframe/Example.svelte

```diff
@@ -5,8 +5,7 @@
 	export let index: number;
 
 	let hovered = false;
-	let loaded_value: (string | number)[][] | string = value;
-	let loaded = Array.isArray(loaded_value);
```

Comment:
I'm not sure why we added `loaded_value`. It has no effect except to make `value` non-reactive -- i.e. updating `value` won't update the component. The [original PR](https://github.com/gradio-app/gradio/commit/6b77ea07d307be0dc55ff29f09ae9b64e133c72b) suggests that this was needed to show dataframe examples correctly when they are file names, however, this is not needed as can be confirmed with a simple example:

```py
import gradio as gr
import numpy as np

data = np.random.random((100,5))
np.savetxt('test.csv', data, fmt = '%.2f', delimiter=',', header = 'c1, c2, c3, c4, c5')

gr.Interface(lambda x:x, "dataframe", "dataframe", examples=["test.csv", "test.csv"]).launch()
```

Reply:
Lol no clue who would ever write that 🙈